HPE3PARPSToolkit-CLI.psm1

```diff
@@ -34380,11 +34379,2491 @@ Function Stop-3parWsapi()
+##///////////////////////////////////////////////////////////////
+##///////////////////////////////////////////////////////////////
+##v2.3 
+##///////////////////////////////////////////////////////////////
+##///////////////////////////////////////////////////////////////
+
+##########################################################################
+######################### FUNCTION Set-3parDomain #########################
+##########################################################################
+Function Set-3parDomain()
+{
+<#
+  .SYNOPSIS
+   Set-3parDomain Change currentdomain CLI environment parameter.
+
+  .DESCRIPTION
+   The Set-3parDomain command changes the currentdomain CLI environment parameter.
+
+  .EXAMPLE
+   Set-3parDomain
+   
+  .EXAMPLE
+   Set-3parDomain -Domain "XXX"
+   
+  .PARAMETER Domain
+	Name of the domain to be set as the working domain for the current CLI session.  
+	If the <domain> parameter is not present or is equal to -unset then the working domain is set to no current domain.
+	
+
+  .Notes
+    NAME: Set-3parDomain
+    LASTEDIT 14-03-2019 14:24:44
+    KEYWORDS: 3parVersion
```

Comment:
Why only 3parVersion given as the KEYWORDS. As per my view it should show this PS cmdlet as the Keyword.

Reply:
Yes,Keyword should be Cmdlet Name. I updated with the same.